### PR TITLE
fixed diff on last slash of uri

### PR DESCRIPTION
**Pull request by edwardmedia**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/11977

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
cloudscheduler: fixed a diff on the last slash of uri on `google_cloud_scheduler_job`
```

### CODE REVIEW

File: mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb [hunk 1]
```diff
@@ -285,3 +285,37 @@ func TestDurationDiffSuppress(t *testing.T) {
 		}
 	}
 }
+
+func TestLastSlashDiffSuppress(t *testing.T) {
+	cases := map[string]struct {
+		Old, New           string
+		ExpectDiffSuppress bool
+	}{
+		"locations to zones": {
```
Comment (megan07) [inline mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb:294]: Do we want to change the names of these tests? I'm assuming they were copied and pasted :)

File: mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb [hunk 2]
```diff
@@ -285,3 +285,37 @@ func TestDurationDiffSuppress(t *testing.T) {
...
+		"locations to locations": {
+			Old:                "https://hello-rehvs75zla-uc.a.run.app/",
+			New:                "https://hello-rehvs75zla-uc.a.run.app/",
+			ExpectDiffSuppress: true,
+		},
+		"zones to regions": {
+			Old:                "https://x.a.run.app/",
+			New:                "https://y.a.run.app",
+			ExpectDiffSuppress: false,
+		},
```
Comment (megan07) [inline mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb:313]: Do we want to add a test where neither end in a slash? Just to cover everything? Or is that overkill?

**Review by megan07 — CHANGES_REQUESTED** [review]

File: mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb (hunk 1 above)
Comment (edwardmedia) [reply mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb]: resolved

File: mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb (hunk 2 above)
Comment (edwardmedia) [reply mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb:318]: added

File: mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb [hunk 3]
```diff
@@ -285,3 +285,42 @@ func TestDurationDiffSuppress(t *testing.T) {
...
+			Old:                "https://hello-rehvs75zla-uc.a.run.app",
+			New:                "https://hello-rehvs75zla-uc.a.run.app/",
+			ExpectDiffSuppress: true,
+		},
+		"slash to slash": {
+			Old:                "https://hello-rehvs75zla-uc.a.run.app/",
+			New:                "https://hello-rehvs75zla-uc.a.run.app/",
+			ExpectDiffSuppress: true,
+		},
+		"no slash to no slash": {
```
Comment (megan07) [inline mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb:309]: I think you forgot to remove the slashes here :)

**Review by megan07 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**